### PR TITLE
git: Ignore services.log

### PR DESCRIPTION
**Pull request by def-**
Follow-up to https://github.com/MaterializeInc/materialize/pull/22367

### Checklist

- [ ] This PR has adequate test coverage / QA involvement has been duly considered.
- [ ] This PR has an associated up-to-date [design doc](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/README.md), is a design doc ([template](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/00000000_template.md)), or is sufficiently small to not require a design.
  
- [ ] If this PR evolves [an existing `$T ⇔ Proto$T` mapping](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/command-and-response-binary-encoding.md) (possibly in a backwards-incompatible way), then it is tagged with a `T-proto` label.
- [ ] If this PR will require changes to cloud orchestration or tests, there is a companion cloud PR to account for those changes that is tagged with the release-blocker label ([example](https://github.com/MaterializeInc/cloud/pull/5021)).
  
- [ ] This PR includes the following [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note):
  -

### CODE REVIEW

**Review by nrainer-materialize — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -25,13 +25,15 @@ test/feature-benchmark/tmp/*.td
...
 junit_*.xml
+parallel-workload-queries.log
 perf.data*
 **/*.rej
 **/*.orig
 **/*.profraw
 **/*.profdata
 **/*.debug
 scratch
+services.log
```
Comment (nrainer-materialize) [inline .gitignore:36]: Do you happen to know where the logging path is specified? When running scalability tests from `tests/scalability`, I get the `services.log` file added to the repo root.

File: services.log
Comment (philip-stoev) [inline services.log:1]: So you are deleting an already-committed services.log

**Review by philip-stoev — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (def-) [reply .gitignore:36]: Actually it's the opposite. `**/` prefix is useless. I removed it from all other .gitignore entries too.

File: services.log
Comment (def-) [reply services.log:1]: Correct, that's how I got the idea for this .gitignore.

**Review by philip-stoev — APPROVED** [review]

**Pull request merged**